addons/account/static/src/components/autosave_many2many_tax_tags/autosave_many2many_tax_tags.js

```diff
@@ -1,8 +1,11 @@
+/** @odoo-module **/
+
 import { registry } from "@web/core/registry";
 import { useRecordObserver } from "@web/model/relational_model/utils";
-import { Many2ManyTagsField, many2ManyTagsField } from "@web/views/fields/many2many_tags/many2many_tags_field";
+import { many2ManyTagsField } from "@web/views/fields/many2many_tags/many2many_tags_field";
+import { Many2ManyTaxTagsField } from "@account/components/many2x_tax_tags/many2x_tax_tags";
```

Comment:
I think you can import `many2ManyTaxTagsField` also instead. (If I'm not mistaken, you could keep importing to extend the same 2 before or the 2 from account. Here you're doing half of each)